generator/views/view.py

```diff
@@ -3,6 +3,8 @@
 
 from typing import Any, Dict, Iterator, List, Optional, TypedDict
 
+import click
```

Comment:
Maybe just import the exception class that you need here, for clarity?

```suggestion
from click import ClickException
```